### PR TITLE
HPCC-17673 DOCS:Update ECL Watch Landing Zone images

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17673 DOCS:Update ECL Watch Landing Zone images
Update the Landing Zone section (in the ECL Watch) document.
Also affected other docs:  Data Handling, Data Tutorial, IMDB,
Installing and Running, Instant Cloud, One Click, AWS, and VM.

@miguelvazq please review w/ special regards to the images 
@JamesDeFabia please review

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
The images look fine.

File: docs/HPCCDataHandling/DH-Mods/DH-Mod1.xml [hunk 1]
```diff
@@ -100,17 +100,16 @@
               </mediaobject>
             </figure></para>
 
-          <para><phrase> </phrase>Once you click on the Upload/download file
-          link, it will take you to the dropzones and files page, where you
-          can choose to <emphasis role="bold">Browse</emphasis> your machine
-          for a file to upload:</para>
+          <para><phrase> </phrase>Once you click on the Upload file link, it
```
Comment (JamesDeFabia) [inline docs/HPCCDataHandling/DH-Mods/DH-Mod1.xml]: Check the workflow. When you click, you get a system file dialog.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
One comment inline

**Comment by g-pan** [comment]
@JamesDeFabia please review

File: docs/HPCCDataHandling/DH-Mods/DH-Mod1.xml [hunk 2]
```diff
@@ -98,36 +101,31 @@
                              vendor="eclwatchSS" />
                 </imageobject>
               </mediaobject>
-            </figure></para>
-
-          <para><phrase> </phrase>Once you click on the Upload/download file
-          link, it will take you to the dropzones and files page, where you
-          can choose to <emphasis role="bold">Browse</emphasis> your machine
-          for a file to upload:</para>
+            </figure>A system file dialog opens. Browse to and select the file
```
Comment (JamesDeFabia) [inline docs/HPCCDataHandling/DH-Mods/DH-Mod1.xml]: Browse to and select the file
s/b 
Browse to select the file

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
One minor comment

**Comment by g-pan** [comment]
@JamesDeFabia please review

**Review by JamesDeFabia — APPROVED** [review]
Good to merge

**Pull request merged**